--- arabica-2004-february/examples/SAX/Makefile~	2003-09-10 15:34:07.000000000 +0200
+++ arabica-2004-february/examples/SAX/Makefile	2005-06-12 14:02:37.840087048 +0200
@@ -7,7 +7,7 @@
 LIBS_DIRS += -L../../bin 
 
 STATIC_LIBS +=
-DYNAMIC_LIBS += -lArabica
+MORE_LIBS = $(DYNAMIC_LIBS) -lArabica
 
 ######################################
 # SOURCE FILES
@@ -28,15 +28,15 @@
 all : pyx simple_handler writer
 
 pyx$ : $(PYX_OBJS)
-	$(LD) $(LDFLAGS) -o $@ $(PYX_OBJS) $(LIBS_DIRS) $(STATIC_LIBS) $(DYNAMIC_LIBS)
+	$(LD) $(LDFLAGS) -o $@ $(PYX_OBJS) $(LIBS_DIRS) $(STATIC_LIBS) $(MORE_LIBS)
 	$(COPY) pyx$(EXESUFFIX) ../../bin
 
 simple_handler : $(SIMPLE_OBJS)
-	$(LD) $(LDFLAGS) -o $@ $(SIMPLE_OBJS) $(LIBS_DIRS) $(STATIC_LIBS) $(DYNAMIC_LIBS)
+	$(LD) $(LDFLAGS) -o $@ $(SIMPLE_OBJS) $(LIBS_DIRS) $(STATIC_LIBS) $(MORE_LIBS)
 	$(COPY) simple_handler$(EXESUFFIX) ../../bin
 
 writer : $(WRITER_OBJS)
-	$(LD) $(LDFLAGS) -o $@ $(WRITER_OBJS) $(LIBS_DIRS) $(STATIC_LIBS) $(DYNAMIC_LIBS)
+	$(LD) $(LDFLAGS) -o $@ $(WRITER_OBJS) $(LIBS_DIRS) $(STATIC_LIBS) $(MORE_LIBS)
 	$(COPY) writer$(EXESUFFIX) ../../bin
 
 ######################
--- arabica-2004-february/examples/SAX2DOM/Makefile~	2003-09-10 15:34:07.000000000 +0200
+++ arabica-2004-february/examples/SAX2DOM/Makefile	2005-06-12 14:09:43.491378216 +0200
@@ -7,7 +7,7 @@
 LIBS_DIRS += -L../../bin 
 
 STATIC_LIBS +=
-DYNAMIC_LIBS += -lArabica
+MORE_LIBS = $(DYNAMIC_LIBS) -lArabica
 
 ######################################
 # SOURCE FILES
@@ -32,11 +32,11 @@
 all : sax2dom_test domwriter
 
 sax2dom_test : $(SAXDOM_OBJS) $(CPPUNITFW_OBJS) $(CPPUNITUI_OBJS)
-	$(LD) $(LDFLAGS) -o $@ $(SAXDOM_OBJS) $(CPPUNITFW_OBJS) $(CPPUNITUI_OBJS) $(LIBS_DIRS) $(STATIC_LIBS) $(DYNAMIC_LIBS)
+	$(LD) $(LDFLAGS) -o $@ $(SAXDOM_OBJS) $(CPPUNITFW_OBJS) $(CPPUNITUI_OBJS) $(LIBS_DIRS) $(STATIC_LIBS) $(MORE_LIBS)
 	$(COPY) sax2dom_test$(EXESUFFIX) ../../bin
 
 domwriter : $(DOMWRITER_OBJS)
-	$(LD) $(LDFLAGS)  -o $@ $(DOMWRITER_OBJS) $(LIBS_DIRS) $(STATIC_LIBS) $(DYNAMIC_LIBS)
+	$(LD) $(LDFLAGS)  -o $@ $(DOMWRITER_OBJS) $(LIBS_DIRS) $(STATIC_LIBS) $(MORE_LIBS)
 	$(COPY) domwriter$(EXESUFFIX) ../../bin
 
 ######################
--- arabica-2004-february/examples/Utils/Makefile~	2005-06-12 15:04:08.797976160 +0200
+++ arabica-2004-february/examples/Utils/Makefile	2005-06-12 15:03:59.181438096 +0200
@@ -7,7 +7,7 @@
 LIBS_DIRS += -L../../bin 
 
 STATIC_LIBS +=
-DYNAMIC_LIBS += -lArabica
+MORE_LIBS = $(DYNAMIC_LIBS) -lArabica
 
 ######################################
 # SOURCE FILES
@@ -20,7 +20,7 @@
 all : transcode
 
 transcode : $(TRANSCODE_OBJS)
-	$(LD) $(LDFLAGS) -o $@ $(TRANSCODE_OBJS) $(LIBS_DIRS) $(STATIC_LIBS) $(DYNAMIC_LIBS)
+	$(LD) $(LDFLAGS) -o $@ $(TRANSCODE_OBJS) $(LIBS_DIRS) $(STATIC_LIBS) $(MORE_LIBS)
 	$(COPY) transcode$(EXESUFFIX) ../../bin
 
 
